### ISSUE TITLE
Navbar Layout and Appearance Fix

### ISSUE THREAD

**Issue by amitb0ra** [Avdhesh-Varshney/chanakya-niti-backend#10]
The background color and text color of the navbar  lacks sufficient contrast, which may affect accessibility and readability.

![image](https://github.com/user-attachments/assets/4a48234b-485a-474e-9b9c-659c7bf91053)

Suggested Fixes:
Adjust the background color or text color of both the navbar and the legend section to ensure high readability and compliance with accessibility standards

@Avdhesh-Varshney Please assign this to me under gssoc-ext and hacktoberfest
